tests/util/constants.ts

```diff
@@ -37,6 +37,7 @@ export const DEFAULT_GENESIS_STAKING = 1_000n * GLMR;
 export const DEFAULT_GENESIS_MAPPING = 100n * GLMR;
 export const MIN_GLMR_STAKING = 1000n * GLMR;
 export const MIN_GLMR_NOMINATOR = 5n * GLMR;
+export const MIN_GLMR_NOMINATOR_PLUS_ONE = 6n * GLMR;
```

Comment:
Do we really need that constant ?

Reply:
Not really but it's a case of readability vs conciseness. Do you think I should just hard code the variable in the test instead?